spec/formatting.md

```diff
@@ -428,6 +432,32 @@ These are divided into the following categories:
     > when * {The value is not one.}
     > ```
 
+  - **Variable Redefinition errors** occur when
+    the left-hand side of a variable declaration is already defined,
```

Comment:
Get rid of "left-hand side". The formal name `variable` and the term `definition` work fine.

```suggestion
a `declaration` contains a `variable` that is already defined
```